pkgs/development/python-modules/uvloop/default.nix

```diff
@@ -9,16 +9,19 @@
 
 buildPythonPackage rec {
   pname = "uvloop";
-  version = "0.12.2";
+  version = "0.13.0";
   disabled = isPy27;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c48692bf4587ce281d641087658eca275a5ad3b63c78297bbded96570ae9ce8f";
+    sha256 = "0blcnrd5vky2k1m1p1skx4516dr1jx76yyb0c6fi82si6mqd0b4l";
   };
 
   buildInputs = [ libuv ];
 
+  # The flake8 config file is not bundled with the pypi release, but
```

Comment:
Why not use a github release https://github.com/MagicStack/uvloop/releases?

Reply:
I was initially considering it, but then I thought it might be preferable to minimize the changes to the derivation. What do you think?

Reply:
if tests are missing it is fine to make the switch and use the GitHUb release. Note, however, that flake8 tests are not interesting for us. It's just linting and style.

Reply:
Yeah, I'm also a bit torn. I was kinda just trying to keep the changes as minimal as possible and stay close to upstream, since I'm not the maintainer. Would you prefer us to try and blacklist this test?

Reply:
unit tests we should try to keep, but these linting/style tests can indeed be blacklisted.

cc @costrouc

Reply:
Alright, so for now I just plainly switched to removing this particular test-file.

Regarding using the github release, I briefly tried it out, but aside from requiring more changes to the derivation, there are some issues with one of the setuptools `build_ext` stages like `error: don't know how to compile C/C++ code on platform 'posix' with '<distutils.unixccompiler.UnixCCompiler object at 0x7ffff71a8a90>' compiler`) - which is odd, as some native compilation steps seem to work just fine. Maybe somebody else knows. (cc @worldofpeace)

So, maybe it's fine to just leave it using `fetchPypi` and ditch the file.